nrf_security/Kconfig

```diff
@@ -1108,7 +1108,6 @@ comment "Advanced configuration section"
 menuconfig NRF_SECURITY_ADVANCED
 	bool
 	prompt "Advanced mbed TLS Configuration Settings"
-	depends on MBEDTLS_VANILLA_BACKEND
```

Comment:
Should those be exposed for all backends now ?
Are all advanced option possibilities valid for cc310 and Oberon, or should there maybe still be some filtering with `depends on` ?

Reply:
There are filtering on all items that are limited to the vanilla backend already....
But we need to have a way to set the supportes ciphersuites